ffffm-button-bind/files/etc/rc.button/rfkill.btnb

```diff
@@ -2,27 +2,47 @@
 
 BIND=$(uci get button-bind.wifi.function)
 
+TMP=/tmp/wifi-button-status
```

Comment:
ist gelöst in https://github.com/freifunk-ffm/packages/pull/21/commits/985023d936d18ac13e39016577896c5ffc2db176 `LED_STATE`